app/uk/gov/hmrc/apiplatformoutboundsoap/models/MessageRequest.scala

```diff
@@ -34,7 +34,7 @@ case class Addressing(from: String, to: String, replyTo: String, faultTo: String
   validate(from.trim != "", "addressing.from being empty")
 }
 
-case class PrivateHeader(name: String, value: Option[String]) {
+case class PrivateHeader(name: String, value: String) {
   validate(name.trim.length < 1024, "privateHeaders name is longer than 1024 characters")
-  validate(!(value.isDefined && value.get.trim.length > 1024), "privateHeaders value is longer than 1024 characters")
+  validate(value.trim.length < 1024, "privateHeaders value is longer than 1024 characters")
```

Comment:
Its not a hard requirement to honor the exact number of characters. But it will be useful to keep 1024 for readability instead of changing it to 1025 IMO